scripts/stac/imagery/collection.py

```diff
@@ -308,17 +284,13 @@ def _description(self) -> str:
         elif self.metadata["category"] in [URBAN_AERIAL_PHOTOS, RURAL_AERIAL_PHOTOS]:
             desc = f"Orthophotography within the {region} region captured in the {date} flying season"
         elif self.metadata["category"] == DEM:
-            desc = " ".join(
-                f"Digital Elevation Model within the {region} {geographic_description or ''} region in {date}".split()
-            )
+            desc = f"Digital Elevation Model within the {region} region in {date}"
         elif self.metadata["category"] == DSM:
-            desc = " ".join(
-                f"Digital Surface Model within the {region} {geographic_description or ''} region in {date}".split()
-            )
+            desc = f"Digital Surface Model within the {region} region in {date}"
         else:
             raise SubtypeParameterError(self.metadata["category"])
 
-        if event:
-            desc = desc + f", published as a record of the {event} event"
+        if self.metadata.get("event_name"):
```

Comment:
You could use:
```python
if event_name := self.metadata.get("event_name"):
   desc = desc + f", published as a record of the {event_name} event"
```

to avoid lookup twice. [Example](https://github.com/linz/topo-imagery/blob/dea3189b9afbb198e02888ed6ef9ebb17687a000/scripts/stac/imagery/collection.py#L56).